politeiawww/cmsuser_test.go

```diff
@@ -0,0 +1,247 @@
+package main
+
+import (
+	"encoding/hex"
+	"testing"
+
+	cms "github.com/decred/politeia/politeiawww/api/cms/v1"
+	www "github.com/decred/politeia/politeiawww/api/www/v1"
+	"github.com/decred/politeia/politeiawww/cmd/shared"
+)
+
+func TestInviteNewUser(t *testing.T) {
+	p, cleanup := newTestCMSwww(t)
+	defer cleanup()
+
+	var tests = []struct {
```

Comment:
This is missing some tests.  Typically, you should have tests for every user error that is returned.  This route is a little different since it doesn't return user errors for security reasons, but you should still test every code path that a user can trigger.  

- User is already verified
- User already exists 

Glancing at the route, it looks like those are the only other two paths that could be triggered, but double check that.